src/main/java/hudson/plugins/active_directory/ActiveDirectorySecurityRealm.java

```diff
@@ -488,15 +508,10 @@ public FormValidation doValidate(@QueryParameter(fixEmpty = true) String domain,
          * In a real deployment, often there are servers that don't respond or
          * otherwise broken, so try all the servers.
          */
-        public DirContext bind(String principalName, String password, List<SocketInfo> ldapServers) {
+        public DirContext bind(String principalName, String password, List<SocketInfo> ldapServers, Hashtable<String, String> props) {
             // in a AD forest, it'd be mighty nice to be able to login as "joe"
             // as opposed to "joe@europe",
             // but the bind operation doesn't appear to allow me to do so.
-            Hashtable<String, String> props = new Hashtable<String, String>();
-            props.put(Context.REFERRAL, "follow");
-            props.put("java.naming.ldap.attributes.binary","tokenGroups objectSid");
-            props.put("java.naming.ldap.factory.socket",TrustAllSocketFactory.class.getName());
```

Comment:
again - should not expect a user to have to set this.  not setting it will cause TLS issues.